autoinstrumentation/python/requirements.txt

```diff
@@ -1,56 +1,56 @@
-opentelemetry-distro==0.41b0
+opentelemetry-distro==0.43b0
 # We don't use the distro[otlp] option which automatically includes exporters since gRPC is not appropriate for
 # injected auto-instrumentation, where it has a strict dependency on the OS / Python version the artifact is built for.
-opentelemetry-exporter-otlp-proto-http==1.20.0
+opentelemetry-exporter-otlp-proto-http==1.22.0
 
-opentelemetry-propagator-b3==1.20.0
-opentelemetry-propagator-jaeger==1.20.0
+opentelemetry-propagator-b3==1.22.0
+opentelemetry-propagator-jaeger==1.22.0
 opentelemetry-propagator-aws-xray==1.0.1
 
-opentelemetry-instrumentation==0.41b0
-opentelemetry-propagator-ot-trace==0.41b0
+opentelemetry-instrumentation==0.43b0
+opentelemetry-propagator-ot-trace==0.43b0
 
 # Copied in from https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation
 # except for aws-lambda
-opentelemetry-instrumentation-aio-pika==0.41b0
-opentelemetry-instrumentation-aiohttp-client==0.41b0
-opentelemetry-instrumentation-aiopg==0.41b0
-opentelemetry-instrumentation-asgi==0.41b0
-opentelemetry-instrumentation-asyncpg==0.41b0
-opentelemetry-instrumentation-boto==0.41b0
-opentelemetry-instrumentation-boto3sqs==0.41b0
-opentelemetry-instrumentation-botocore==0.41b0
-opentelemetry-instrumentation-celery==0.41b0
-opentelemetry-instrumentation-confluent-kafka==0.41b0
-opentelemetry-instrumentation-dbapi==0.41b0
-opentelemetry-instrumentation-django==0.41b0
-opentelemetry-instrumentation-elasticsearch==0.41b0
-opentelemetry-instrumentation-falcon==0.41b0
-opentelemetry-instrumentation-fastapi==0.41b0
-opentelemetry-instrumentation-flask==0.41b0
-opentelemetry-instrumentation-grpc==0.41b0
-opentelemetry-instrumentation-httpx==0.41b0
-opentelemetry-instrumentation-jinja2==0.41b0
-opentelemetry-instrumentation-kafka-python==0.41b0
-opentelemetry-instrumentation-logging==0.41b0
-opentelemetry-instrumentation-mysql==0.41b0
-opentelemetry-instrumentation-mysqlclient==0.41b0
-opentelemetry-instrumentation-pika==0.41b0
-opentelemetry-instrumentation-psycopg2==0.41b0
-opentelemetry-instrumentation-pymemcache==0.41b0
-opentelemetry-instrumentation-pymongo==0.41b0
-opentelemetry-instrumentation-pymysql==0.41b0
-opentelemetry-instrumentation-pyramid==0.41b0
-opentelemetry-instrumentation-redis==0.41b0
-opentelemetry-instrumentation-remoulade==0.41b0
-opentelemetry-instrumentation-requests==0.41b0
-opentelemetry-instrumentation-sklearn==0.41b0
-opentelemetry-instrumentation-sqlalchemy==0.41b0
-opentelemetry-instrumentation-sqlite3==0.41b0
-opentelemetry-instrumentation-starlette==0.41b0
-opentelemetry-instrumentation-system-metrics==0.41b0
-opentelemetry-instrumentation-tornado==0.41b0
-opentelemetry-instrumentation-tortoiseorm==0.41b0
-opentelemetry-instrumentation-urllib==0.41b0
-opentelemetry-instrumentation-urllib3==0.41b0
-opentelemetry-instrumentation-wsgi==0.41b0
+opentelemetry-instrumentation-aio-pika==0.43b0
+opentelemetry-instrumentation-aiohttp-client==0.43b0
```

Comment:
Release [0.42b0](https://github.com/open-telemetry/opentelemetry-python-contrib/releases/tag/v0.42b0) added a new package that we should include: opentelemetry-instrumentation-aiohttp-server

Reply:
Just added it.

Reply:
It looks like it didn't get tagged properly: https://github.com/open-telemetry/opentelemetry-python-contrib/tree/v0.43b0/instrumentation.  And it doesnt show up on https://pypi.org/project/opentelemetry-instrumentation-aiohttp-server/.  So lets skip for now.

Reply:
https://github.com/open-telemetry/opentelemetry-python-contrib/issues/2053

Reply:
Lets remove this package from our requirements.txt again.  Hopefully they fix the tagging in the future.